components/Score/Score.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from "react";
+import styled from "styled-components";
+
+export default function Score({ score }) {
+  const [color, setColor] = useState("orange");
+  useEffect(() => {
```

Comment:
hey Florian, sicher dass du den Use Effect hier brauchst? Vielleicht erschließt es sich aber mir nur nicht so ganz....

Reply:
Resultat => Error: Too many re-renders. React limits the number of renders to prevent an infinite loop.
useEffect löst hier nur aus wenn sich >score< ändert, also ein spiel gewonnen wurde ^^
Sonst müsste ich eine if/else -Abfrage machen oder checkColor in der hauptdatei aufrufen,
wobei das glaube ich viel cleaner in der score.js verwaltet wird :D